README.md

```diff
@@ -6,9 +6,10 @@ This action to install and configure Koyeb CLI.
 
 ### Inputs
 
-| Input              | Required | Description                                                      |
-|--------------------|:--------:|------------------------------------------------------------------|
-| api_token          | [x]      | Your Koyeb API token. See https://app.koyeb.com/account/profile. |
+| Input              | Required | Description                                                         |
+|--------------------|:--------:|---------------------------------------------------------------------|
+| api_token          | [x]      | Your Koyeb API token. See https://app.koyeb.com/account/profile.    |
+| github_token       | [x]      | A github API token. It automatically exists as secrets.GITHUB_TOKEN |
```

Comment:
I don't think it's best to advise users to give us their github token, as it has too broad authorizations (if we can directly give the scope needed, it would be even better, I think it's `public_repo` but I'm not sure).
```suggestion
| github_token       | [x]      | A github API token allowing to read public repositories (can be read-only) |
```